src/astro/utils/dataframe.py

```diff
@@ -0,0 +1,20 @@
+import pandas as pd
+
+from astro.constants import ColumnCapitalization
+
+
+def convert_dataframe_col_case(
```

Comment:
Since the function name is already quite long, it may be better to name it `convert_columns_names_capitalization` - this way, we are consistent with the argument name "columns_names_capitalization".

Reply:
Done.